app/views/clients/_clients.html.erb

```diff
@@ -3,6 +3,7 @@
   <tr id="<%= dom_id(client) %>" class="<%= client_messages_read_or_unread(client) %>">
     <td><%= link_to client.full_name, client_messages_path(client.id) %></td>
     <td><%= link_to time_ago_in_words(client.contacted_at), client_messages_path(client.id) %></td>
+    <td><%= link_to 'Edit', edit_client_path(client) %></td>
```

Comment:
Instead of the word "Edit", can we use the settings icon GCF uses (`<i class="icon-settings"></i>`)?